nerd/payload/task.go

```diff
@@ -0,0 +1,17 @@
+package payload
```

Comment:
The payload structure is gonna be in the form of: TaskCreateInput / TaskCreateOutput. for example: https://github.com/nerdalize/universe/blob/fix-cleantasks/server/payload/task.go